pkg/v1/providers/infrastructure-vsphere/yttcc/vsphere-overlay.yaml

```diff
@@ -22,15 +22,4 @@ spec:
       machineDeployments:
       #@overlay/match by=overlay.index(0)
       - replicas: #@ data.values.WORKER_MACHINE_COUNT_0
```

Comment:
I think we should overwrite md-0's replica with WORKER_MACHINE_COUNT_0 in pkg/v1/providers/infrastructure-vsphere/v1.3.1/yttcc/overlay.yaml.

Reply:
you mean remove this code completely and move this logic to `pkg/v1/providers/infrastructure-vsphere/v1.3.1/yttcc/overlay.yaml` ? I can change that, but it seems that this file cannot be removed (will cause error since it's expected to exist in another template file), I will remove the logic here and keep the file

Reply:
code and description updated.